src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -653,23 +657,28 @@ static inline int MPIDI_NM_mpi_get(void *origin_addr,
                                    MPIDI_av_entry_t * addr)
 {
     int mpi_errno = MPI_SUCCESS;
+
```

Comment:
What's this white space change?  Doesn't look like a reformatting required for consistency.